dashboard/app/views/layouts/application.html.haml

```diff
@@ -1,6 +1,11 @@
 !!! 5
 %html{dir: locale_dir}
   %head
+    - if request.locale === "in-TL"
+      :javascript
+        var _jipt = [];
+        _jipt.push(['project', 'codeorg']);
+      %script{type: "text/javascript", src: "//cdn.crowdin.com/jipt/jipt.js"}
```

Comment:
Could you add a quick comment here which links to https://crowdin.com/project/codeorg/settings#in-context so future developers kind learn more about what we are doing here? Maybe also add a comment that `in-TL` is a pseudo-language locale used for supporting the Crowdin In-Context translation tool?